### PR TITLE
Manage Profile linked to updateUserAccountInfo

### PR DESCRIPTION
**Pull request by emrb9**
Manage Profile(frontend) Linked to updateUserAccountInfo (backend)

### CODE REVIEW

File: frontend/src/Components/Pages/ManageProfilePage.js [hunk 1]
```diff
@@ -22,60 +74,58 @@ function ManageProfilePage() {
...
+                    </Form>
+                    <div className='d-grid'>
+                            <Button variant="primary" type="submit" onClick={handleSaveChanges}>
                                 Save Changes
                             </Button>   
                         </div>
-                    </Form>
                 </Card.Body>
             </Card>
+            <button onClick={() => console.log(handleSaveChanges) }>hello</button>
```
Comment (nolanbastien) [inline frontend/src/Components/Pages/ManageProfilePage.js:128]: What is this button again ? Is this the button to test something ?

**Review by nolanbastien — CHANGES_REQUESTED** [review]
Great job all around !

File: frontend/src/Components/Pages/ManageProfilePage.js (hunk 1 above)
Comment (emrb9) [reply frontend/src/Components/Pages/ManageProfilePage.js]: It was the test button. I am removing it now :)

**Review by Kacocs — COMMENTED** [review]
Its nice to see the different ways we can link it to backend. 
The code is good and ready to merge. 
Good work!


Just for reference:  I worked with Emily on this task.

**Comment by emrb9** [comment]
https://we.tl/t-09adTyP7it

**Pull request merged**